doc/quickstart/postgis_quickstart.rst

```diff
@@ -224,8 +226,8 @@ We'll be focussing our attention on that type.
 
 
 
-Creating A Spatial Table The Hard Way
-================================================================================
+Creating a spatial table the hard way
```

Comment:
I understand why this is important, but it seems off-putting in a "quickstart" document. Perhaps this could be moved further down to the "Things to try" section which is a bit more of a challenge.?

Reply:
My feeling is we should get rid of the word "Hard"  and revise the title to 

Creating a Spatial Table using SQL

As far as moving it down.  No - because honestly I don't find QGIS easier.  In fact since users are already in pgAdmin or psql, all they need to do is "Copy and paste the text"

If you move up then you've got to explain QGIS to them and then oh those click here click here and here.  Not any easier.

The QGIS one should be:

Creating a Spatial Table via Data Loading (QGIS).

Reply:
I like the suggestion to change the title: Creating a Spatial Table using SQL

And please leave it here and not further down

Reply:
I revised the title as suggested. I think there's two differing suggestions for the title for the QGIS section. "Creating a Spatial Table via Data Loading (QGIS)" or "Importing spatial data into the database". I've implemented the second suggestion but you can review again to see what you think.

Reply:
@astroidex Are we able to merge this PR now? Cameron and I are going through a process to wrap things up in readiness for the docs-only release. Let me know. Thanks